optuna/cli.py

```diff
@@ -382,6 +382,30 @@ def take_action(self, parsed_args: Namespace) -> int:
         return 0
 
 
+class _StudyNames(_BaseCommand):
+    """Get all study names stored in a specified storage"""
+
+    def add_arguments(self, parser: ArgumentParser) -> None:
+        parser.add_argument(
+            "-f",
+            "--format",
+            type=str,
+            choices=("json", "table", "yaml"),
+            default="table",
+            help="Output format.",
+        )
```

Comment:
How about removing format option and just printing study names per line?

Reply:
We discussed internally and decided to provide format option. Let me resolve this.